debian/changelog

```diff
@@ -0,0 +1,5 @@
+libkiwix (0.0.0) unstable; urgency=medium
```

Comment:
Souldn't be the version of kiwix-lib ?

Reply:
No, this is just a placeholder. The real version is either taken from the tag being pushed or extracted out of meson.build (see https://github.com/legoktm/gh-action-auto-dch/blob/master/auto_dch.py#L29). In theory we could put the real version here, but then it would be another thing for y'all to bump and update when making a new release, so I tried to avoid that.